### DISCUSSION TITLE
how to stop "File Name" translation

### DISCUSSION THREAD

**Discussion by Abu-omr** [bookfere/Ebook-Translator-Calibre-Plugin#314] [General]
I applaud you for this most wonderful program. I have noticed the development of the program in recent times... and its clear support for local languages. But there is a problem: the program translates file names into the local language, but most of the names are not correct due to automatic translation. It is better for this option to work or Stops if it is important to some users.

**Comment by bookfere** [comment]
The requirement has been resolved in #301. Please install the __[Rolling Release](https://translator.bookfere.com/)__ until the next version is released.
